src/arrays/why-choose-us.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
there's an export on line 5 and line 32

You can do it either way, but the one at the bottom complies with eslint's default export rule

Most important thing is to remember to either add or take away {} on  import based on the one you choose

Reply:
Ohh good catch, I'll remove the redundant ones.